Resources/fsh-generated/resources/CapabilityStatement-ISiK-capabilityStatement-dokumentenaustausch-server.json

```diff
@@ -6,7 +6,7 @@
   "experimental": false,
   "version": "3.0.0",
   "publisher": "gematik GmbH",
-  "date": "2023-07-01",
+  "date": "2023-11-28",
```

Comment:
Datum würde ich nur bei Inkrementierung ändern. Dann aber auch als TC mit Releasenotes etc.

Reply:
`date` gibt an, wann das Profil zuletzt ein Update erfahren hat. Da wir in diesem Fall Links fixen, würde ich es setzen.